network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/extensions/BatteryShortCircuitImpl.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.powsybl.network.store.iidm.impl.extensions;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.commons.extensions.AbstractExtension;
+import com.powsybl.iidm.network.Battery;
+import com.powsybl.iidm.network.extensions.BatteryShortCircuit;
+import com.powsybl.network.store.iidm.impl.BatteryImpl;
+
+/**
+ * @author Etienne Lesot <etienne.lesot at rte-france.com>
+ */
+public class BatteryShortCircuitImpl extends AbstractExtension<Battery> implements BatteryShortCircuit {
```

Comment:
It's a pity to copy all the implementation of GeneratorSHortCircuitImpl.
Powsybl-core implementation has a common class: https://github.com/powsybl/powsybl-core/blob/182120e3ec2f6c8bd1a1df86bd4b1e2c490d2692/iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/extensions/AbstractShortCircuitExtensionImpl.java#L20

Reply:
done